[2024-08-11 00:03:38,621] {scheduler_job.py:182} INFO - Started process (PID=676) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:03:38,621] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 00:03:38,622] {logging_mixin.py:104} INFO - [2024-08-11 00:03:38,622] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:03:38,631] {logging_mixin.py:104} INFO - [2024-08-11 00:03:38,629] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 00:03:38,632] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:03:38,691] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 00:04:09,433] {scheduler_job.py:182} INFO - Started process (PID=705) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:04:09,434] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 00:04:09,434] {logging_mixin.py:104} INFO - [2024-08-11 00:04:09,434] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:04:09,441] {logging_mixin.py:104} INFO - [2024-08-11 00:04:09,440] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 00:04:09,442] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 00:04:09,501] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 01:04:35,176] {scheduler_job.py:182} INFO - Started process (PID=734) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:04:35,177] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 01:04:35,179] {logging_mixin.py:104} INFO - [2024-08-11 01:04:35,179] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:04:35,193] {logging_mixin.py:104} INFO - [2024-08-11 01:04:35,191] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 01:04:35,194] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:04:35,270] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.097 seconds
[2024-08-11 01:05:06,301] {scheduler_job.py:182} INFO - Started process (PID=763) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:05:06,302] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 01:05:06,302] {logging_mixin.py:104} INFO - [2024-08-11 01:05:06,302] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:05:06,312] {logging_mixin.py:104} INFO - [2024-08-11 01:05:06,309] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 01:05:06,313] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 01:05:06,372] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 02:05:32,580] {scheduler_job.py:182} INFO - Started process (PID=792) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:05:32,585] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 02:05:32,586] {logging_mixin.py:104} INFO - [2024-08-11 02:05:32,586] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:05:32,594] {logging_mixin.py:104} INFO - [2024-08-11 02:05:32,593] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 02:05:32,595] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:05:32,670] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.092 seconds
[2024-08-11 02:06:02,861] {scheduler_job.py:182} INFO - Started process (PID=821) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:06:02,862] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 02:06:02,862] {logging_mixin.py:104} INFO - [2024-08-11 02:06:02,862] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:06:02,869] {logging_mixin.py:104} INFO - [2024-08-11 02:06:02,868] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 02:06:02,870] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 02:06:02,927] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.068 seconds
[2024-08-11 03:06:29,519] {scheduler_job.py:182} INFO - Started process (PID=850) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:29,521] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 03:06:29,522] {logging_mixin.py:104} INFO - [2024-08-11 03:06:29,522] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:29,533] {logging_mixin.py:104} INFO - [2024-08-11 03:06:29,532] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 03:06:29,536] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:29,606] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.088 seconds
[2024-08-11 03:06:59,683] {scheduler_job.py:182} INFO - Started process (PID=879) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:59,684] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 03:06:59,684] {logging_mixin.py:104} INFO - [2024-08-11 03:06:59,684] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:59,692] {logging_mixin.py:104} INFO - [2024-08-11 03:06:59,691] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 03:06:59,693] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 03:06:59,752] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 04:07:27,587] {scheduler_job.py:182} INFO - Started process (PID=908) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:27,588] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 04:07:27,589] {logging_mixin.py:104} INFO - [2024-08-11 04:07:27,589] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:27,609] {logging_mixin.py:104} INFO - [2024-08-11 04:07:27,602] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 04:07:27,612] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:27,685] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.100 seconds
[2024-08-11 04:07:58,641] {scheduler_job.py:182} INFO - Started process (PID=937) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:58,641] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 04:07:58,642] {logging_mixin.py:104} INFO - [2024-08-11 04:07:58,642] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:58,651] {logging_mixin.py:104} INFO - [2024-08-11 04:07:58,650] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 04:07:58,651] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:07:58,711] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 04:55:10,713] {scheduler_job.py:182} INFO - Started process (PID=966) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:55:10,715] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 04:55:10,716] {logging_mixin.py:104} INFO - [2024-08-11 04:55:10,716] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:55:10,730] {logging_mixin.py:104} INFO - [2024-08-11 04:55:10,725] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 04:55:10,732] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 04:55:10,803] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.092 seconds
[2024-08-11 05:08:54,921] {scheduler_job.py:182} INFO - Started process (PID=995) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:08:54,923] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 05:08:54,923] {logging_mixin.py:104} INFO - [2024-08-11 05:08:54,923] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:08:54,941] {logging_mixin.py:104} INFO - [2024-08-11 05:08:54,940] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 05:08:54,942] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:08:55,007] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.092 seconds
[2024-08-11 05:43:27,767] {scheduler_job.py:182} INFO - Started process (PID=1024) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:43:27,770] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 05:43:27,770] {logging_mixin.py:104} INFO - [2024-08-11 05:43:27,770] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:43:27,783] {logging_mixin.py:104} INFO - [2024-08-11 05:43:27,782] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 05:43:27,784] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 05:43:27,848] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.083 seconds
[2024-08-11 06:09:51,965] {scheduler_job.py:182} INFO - Started process (PID=1053) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:09:51,966] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 06:09:51,968] {logging_mixin.py:104} INFO - [2024-08-11 06:09:51,968] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:09:51,976] {logging_mixin.py:104} INFO - [2024-08-11 06:09:51,975] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 06:09:51,977] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:09:52,037] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 06:39:54,458] {scheduler_job.py:182} INFO - Started process (PID=1082) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:39:54,458] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 06:39:54,459] {logging_mixin.py:104} INFO - [2024-08-11 06:39:54,459] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:39:54,468] {logging_mixin.py:104} INFO - [2024-08-11 06:39:54,467] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 06:39:54,468] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:39:54,527] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 06:40:24,635] {scheduler_job.py:182} INFO - Started process (PID=1111) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:40:24,636] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 06:40:24,636] {logging_mixin.py:104} INFO - [2024-08-11 06:40:24,636] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:40:24,647] {logging_mixin.py:104} INFO - [2024-08-11 06:40:24,646] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 06:40:24,648] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 06:40:24,706] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 07:00:21,812] {scheduler_job.py:182} INFO - Started process (PID=1140) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:21,813] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 07:00:21,816] {logging_mixin.py:104} INFO - [2024-08-11 07:00:21,816] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:21,826] {logging_mixin.py:104} INFO - [2024-08-11 07:00:21,824] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 07:00:21,827] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:21,935] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.124 seconds
[2024-08-11 07:00:52,767] {scheduler_job.py:182} INFO - Started process (PID=1169) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:52,769] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 07:00:52,769] {logging_mixin.py:104} INFO - [2024-08-11 07:00:52,769] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:52,777] {logging_mixin.py:104} INFO - [2024-08-11 07:00:52,776] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 07:00:52,777] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:00:52,836] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 07:10:51,126] {scheduler_job.py:182} INFO - Started process (PID=1198) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:10:51,127] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 07:10:51,129] {logging_mixin.py:104} INFO - [2024-08-11 07:10:51,129] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:10:51,156] {logging_mixin.py:104} INFO - [2024-08-11 07:10:51,155] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 07:10:51,159] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:10:51,233] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.110 seconds
[2024-08-11 07:27:48,938] {scheduler_job.py:182} INFO - Started process (PID=1227) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:27:48,939] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 07:27:48,939] {logging_mixin.py:104} INFO - [2024-08-11 07:27:48,939] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:27:48,947] {logging_mixin.py:104} INFO - [2024-08-11 07:27:48,946] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 07:27:48,947] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:27:49,006] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.069 seconds
[2024-08-11 07:47:28,719] {scheduler_job.py:182} INFO - Started process (PID=1256) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:47:28,720] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 07:47:28,720] {logging_mixin.py:104} INFO - [2024-08-11 07:47:28,720] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:47:28,730] {logging_mixin.py:104} INFO - [2024-08-11 07:47:28,727] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 07:47:28,731] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 07:47:28,790] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 08:11:31,289] {scheduler_job.py:182} INFO - Started process (PID=1285) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:11:31,289] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 08:11:31,290] {logging_mixin.py:104} INFO - [2024-08-11 08:11:31,289] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:11:31,297] {logging_mixin.py:104} INFO - [2024-08-11 08:11:31,296] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 08:11:31,298] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:11:31,357] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 08:12:02,089] {scheduler_job.py:182} INFO - Started process (PID=1314) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:12:02,090] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 08:12:02,091] {logging_mixin.py:104} INFO - [2024-08-11 08:12:02,090] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:12:02,101] {logging_mixin.py:104} INFO - [2024-08-11 08:12:02,100] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 08:12:02,102] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 08:12:02,161] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 09:02:46,637] {scheduler_job.py:182} INFO - Started process (PID=1343) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:02:46,638] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 09:02:46,638] {logging_mixin.py:104} INFO - [2024-08-11 09:02:46,638] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:02:46,648] {logging_mixin.py:104} INFO - [2024-08-11 09:02:46,647] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 09:02:46,649] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:02:46,721] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.086 seconds
[2024-08-11 09:12:58,645] {scheduler_job.py:182} INFO - Started process (PID=1372) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:12:58,648] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 09:12:58,649] {logging_mixin.py:104} INFO - [2024-08-11 09:12:58,649] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:12:58,665] {logging_mixin.py:104} INFO - [2024-08-11 09:12:58,660] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 09:12:58,667] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:12:58,776] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.133 seconds
[2024-08-11 09:15:36,521] {scheduler_job.py:182} INFO - Started process (PID=1401) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:15:36,522] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 09:15:36,522] {logging_mixin.py:104} INFO - [2024-08-11 09:15:36,522] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:15:36,531] {logging_mixin.py:104} INFO - [2024-08-11 09:15:36,529] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 09:15:36,532] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:15:36,592] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 09:16:07,624] {scheduler_job.py:182} INFO - Started process (PID=1430) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:16:07,624] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 09:16:07,625] {logging_mixin.py:104} INFO - [2024-08-11 09:16:07,624] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:16:07,632] {logging_mixin.py:104} INFO - [2024-08-11 09:16:07,631] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 09:16:07,633] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 09:16:07,693] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 10:13:39,362] {scheduler_job.py:182} INFO - Started process (PID=1459) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:13:39,363] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 10:13:39,364] {logging_mixin.py:104} INFO - [2024-08-11 10:13:39,364] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:13:39,375] {logging_mixin.py:104} INFO - [2024-08-11 10:13:39,373] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 10:13:39,376] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:13:39,449] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.089 seconds
[2024-08-11 10:14:10,425] {scheduler_job.py:182} INFO - Started process (PID=1488) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:14:10,426] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 10:14:10,426] {logging_mixin.py:104} INFO - [2024-08-11 10:14:10,426] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:14:10,434] {logging_mixin.py:104} INFO - [2024-08-11 10:14:10,433] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 10:14:10,434] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:14:10,493] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 10:52:33,917] {scheduler_job.py:182} INFO - Started process (PID=1517) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:52:33,918] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 10:52:33,920] {logging_mixin.py:104} INFO - [2024-08-11 10:52:33,920] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:52:33,928] {logging_mixin.py:104} INFO - [2024-08-11 10:52:33,927] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 10:52:33,929] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:52:33,991] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 10:53:04,866] {scheduler_job.py:182} INFO - Started process (PID=1546) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:53:04,866] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 10:53:04,866] {logging_mixin.py:104} INFO - [2024-08-11 10:53:04,866] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:53:04,874] {logging_mixin.py:104} INFO - [2024-08-11 10:53:04,873] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 10:53:04,875] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 10:53:04,934] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 11:14:49,986] {scheduler_job.py:182} INFO - Started process (PID=1575) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:14:49,986] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 11:14:49,987] {logging_mixin.py:104} INFO - [2024-08-11 11:14:49,987] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:14:49,996] {logging_mixin.py:104} INFO - [2024-08-11 11:14:49,995] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 11:14:49,997] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:14:50,060] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 11:15:20,474] {scheduler_job.py:182} INFO - Started process (PID=1606) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:15:20,475] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 11:15:20,475] {logging_mixin.py:104} INFO - [2024-08-11 11:15:20,475] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:15:20,483] {logging_mixin.py:104} INFO - [2024-08-11 11:15:20,482] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 11:15:20,483] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 11:15:20,543] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 12:15:46,959] {scheduler_job.py:182} INFO - Started process (PID=1633) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:15:46,960] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:15:46,960] {logging_mixin.py:104} INFO - [2024-08-11 12:15:46,960] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:15:46,969] {logging_mixin.py:104} INFO - [2024-08-11 12:15:46,968] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:15:46,970] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:15:47,029] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:16:17,205] {scheduler_job.py:182} INFO - Started process (PID=1662) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:16:17,205] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:16:17,208] {logging_mixin.py:104} INFO - [2024-08-11 12:16:17,208] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:16:17,215] {logging_mixin.py:104} INFO - [2024-08-11 12:16:17,214] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:16:17,216] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:16:17,274] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 12:19:28,527] {scheduler_job.py:182} INFO - Started process (PID=1691) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:28,528] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:19:28,528] {logging_mixin.py:104} INFO - [2024-08-11 12:19:28,528] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:28,538] {logging_mixin.py:104} INFO - [2024-08-11 12:19:28,537] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:19:28,539] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:28,600] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:19:58,872] {scheduler_job.py:182} INFO - Started process (PID=1721) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:58,873] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:19:58,873] {logging_mixin.py:104} INFO - [2024-08-11 12:19:58,873] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:58,881] {logging_mixin.py:104} INFO - [2024-08-11 12:19:58,880] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:19:58,882] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:19:58,942] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 12:20:29,517] {scheduler_job.py:182} INFO - Started process (PID=1749) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:20:29,517] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:20:29,518] {logging_mixin.py:104} INFO - [2024-08-11 12:20:29,518] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:20:29,527] {logging_mixin.py:104} INFO - [2024-08-11 12:20:29,526] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:20:29,527] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:20:29,587] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:21:00,356] {scheduler_job.py:182} INFO - Started process (PID=1778) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:00,357] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:21:00,357] {logging_mixin.py:104} INFO - [2024-08-11 12:21:00,357] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:00,365] {logging_mixin.py:104} INFO - [2024-08-11 12:21:00,364] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:21:00,366] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:00,426] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:21:31,242] {scheduler_job.py:182} INFO - Started process (PID=1807) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:31,243] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:21:31,243] {logging_mixin.py:104} INFO - [2024-08-11 12:21:31,243] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:31,254] {logging_mixin.py:104} INFO - [2024-08-11 12:21:31,253] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:21:31,254] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:21:31,315] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:22:02,198] {scheduler_job.py:182} INFO - Started process (PID=1836) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:02,199] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:22:02,201] {logging_mixin.py:104} INFO - [2024-08-11 12:22:02,201] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:02,210] {logging_mixin.py:104} INFO - [2024-08-11 12:22:02,209] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:22:02,210] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:02,272] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:22:33,155] {scheduler_job.py:182} INFO - Started process (PID=1865) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:33,156] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:22:33,156] {logging_mixin.py:104} INFO - [2024-08-11 12:22:33,156] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:33,168] {logging_mixin.py:104} INFO - [2024-08-11 12:22:33,167] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:22:33,169] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:22:33,231] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.078 seconds
[2024-08-11 12:23:04,044] {scheduler_job.py:182} INFO - Started process (PID=1894) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:04,045] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:23:04,045] {logging_mixin.py:104} INFO - [2024-08-11 12:23:04,045] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:04,053] {logging_mixin.py:104} INFO - [2024-08-11 12:23:04,052] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:23:04,054] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:04,114] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 12:23:34,794] {scheduler_job.py:182} INFO - Started process (PID=1923) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:34,795] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:23:34,795] {logging_mixin.py:104} INFO - [2024-08-11 12:23:34,795] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:34,805] {logging_mixin.py:104} INFO - [2024-08-11 12:23:34,804] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:23:34,806] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:23:34,867] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:24:04,979] {scheduler_job.py:182} INFO - Started process (PID=1952) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:04,980] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:24:04,980] {logging_mixin.py:104} INFO - [2024-08-11 12:24:04,980] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:04,988] {logging_mixin.py:104} INFO - [2024-08-11 12:24:04,987] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:24:04,988] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:05,048] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 12:24:35,858] {scheduler_job.py:182} INFO - Started process (PID=1981) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:35,858] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:24:35,859] {logging_mixin.py:104} INFO - [2024-08-11 12:24:35,859] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:35,868] {logging_mixin.py:104} INFO - [2024-08-11 12:24:35,867] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:24:35,871] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:24:35,931] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:25:06,704] {scheduler_job.py:182} INFO - Started process (PID=2010) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:06,705] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:25:06,705] {logging_mixin.py:104} INFO - [2024-08-11 12:25:06,705] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:06,714] {logging_mixin.py:104} INFO - [2024-08-11 12:25:06,713] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:25:06,715] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:06,776] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 12:25:37,429] {scheduler_job.py:182} INFO - Started process (PID=2039) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:37,429] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:25:37,430] {logging_mixin.py:104} INFO - [2024-08-11 12:25:37,430] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:37,437] {logging_mixin.py:104} INFO - [2024-08-11 12:25:37,436] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:25:37,438] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:25:37,496] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.069 seconds
[2024-08-11 12:26:08,280] {scheduler_job.py:182} INFO - Started process (PID=2068) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:08,282] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:26:08,282] {logging_mixin.py:104} INFO - [2024-08-11 12:26:08,282] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:08,290] {logging_mixin.py:104} INFO - [2024-08-11 12:26:08,289] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:26:08,291] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:08,353] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 12:26:39,252] {scheduler_job.py:182} INFO - Started process (PID=2097) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:39,254] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:26:39,256] {logging_mixin.py:104} INFO - [2024-08-11 12:26:39,256] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:39,266] {logging_mixin.py:104} INFO - [2024-08-11 12:26:39,263] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:26:39,266] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:26:39,326] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 12:27:10,005] {scheduler_job.py:182} INFO - Started process (PID=2126) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:10,008] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:27:10,008] {logging_mixin.py:104} INFO - [2024-08-11 12:27:10,008] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:10,016] {logging_mixin.py:104} INFO - [2024-08-11 12:27:10,015] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:27:10,017] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:10,076] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:27:40,918] {scheduler_job.py:182} INFO - Started process (PID=2155) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:40,920] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:27:40,922] {logging_mixin.py:104} INFO - [2024-08-11 12:27:40,922] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:40,930] {logging_mixin.py:104} INFO - [2024-08-11 12:27:40,929] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:27:40,934] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:27:40,996] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.080 seconds
[2024-08-11 12:28:11,748] {scheduler_job.py:182} INFO - Started process (PID=2184) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:11,749] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:28:11,749] {logging_mixin.py:104} INFO - [2024-08-11 12:28:11,749] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:11,758] {logging_mixin.py:104} INFO - [2024-08-11 12:28:11,757] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:28:11,760] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:11,820] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 12:28:42,518] {scheduler_job.py:182} INFO - Started process (PID=2213) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:42,518] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:28:42,521] {logging_mixin.py:104} INFO - [2024-08-11 12:28:42,521] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:42,529] {logging_mixin.py:104} INFO - [2024-08-11 12:28:42,528] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:28:42,530] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:28:42,590] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 12:29:13,301] {scheduler_job.py:182} INFO - Started process (PID=2242) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:13,301] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:29:13,304] {logging_mixin.py:104} INFO - [2024-08-11 12:29:13,304] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:13,311] {logging_mixin.py:104} INFO - [2024-08-11 12:29:13,310] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:29:13,312] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:13,371] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:29:44,308] {scheduler_job.py:182} INFO - Started process (PID=2271) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:44,309] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:29:44,311] {logging_mixin.py:104} INFO - [2024-08-11 12:29:44,311] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:44,320] {logging_mixin.py:104} INFO - [2024-08-11 12:29:44,319] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:29:44,322] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:29:44,382] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 12:30:15,044] {scheduler_job.py:182} INFO - Started process (PID=2300) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:15,045] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:30:15,045] {logging_mixin.py:104} INFO - [2024-08-11 12:30:15,045] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:15,054] {logging_mixin.py:104} INFO - [2024-08-11 12:30:15,053] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:30:15,054] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:15,113] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 12:30:46,039] {scheduler_job.py:182} INFO - Started process (PID=2329) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:46,040] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:30:46,040] {logging_mixin.py:104} INFO - [2024-08-11 12:30:46,040] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:46,049] {logging_mixin.py:104} INFO - [2024-08-11 12:30:46,048] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:30:46,049] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:30:46,110] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:31:16,951] {scheduler_job.py:182} INFO - Started process (PID=2358) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:16,952] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:31:16,954] {logging_mixin.py:104} INFO - [2024-08-11 12:31:16,954] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:16,962] {logging_mixin.py:104} INFO - [2024-08-11 12:31:16,961] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:31:16,965] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:17,025] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 12:31:47,723] {scheduler_job.py:182} INFO - Started process (PID=2387) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:47,724] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:31:47,724] {logging_mixin.py:104} INFO - [2024-08-11 12:31:47,724] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:47,733] {logging_mixin.py:104} INFO - [2024-08-11 12:31:47,732] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:31:47,734] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:31:47,794] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 12:32:18,809] {scheduler_job.py:182} INFO - Started process (PID=2416) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:18,810] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:32:18,810] {logging_mixin.py:104} INFO - [2024-08-11 12:32:18,810] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:18,819] {logging_mixin.py:104} INFO - [2024-08-11 12:32:18,818] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:32:18,820] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:18,881] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 12:32:49,705] {scheduler_job.py:182} INFO - Started process (PID=2445) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:49,706] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:32:49,707] {logging_mixin.py:104} INFO - [2024-08-11 12:32:49,707] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:49,717] {logging_mixin.py:104} INFO - [2024-08-11 12:32:49,716] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:32:49,717] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:32:49,777] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 12:33:20,499] {scheduler_job.py:182} INFO - Started process (PID=2474) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:33:20,502] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:33:20,503] {logging_mixin.py:104} INFO - [2024-08-11 12:33:20,503] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:33:20,512] {logging_mixin.py:104} INFO - [2024-08-11 12:33:20,511] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:33:20,513] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:33:20,579] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.082 seconds
[2024-08-11 12:50:13,001] {scheduler_job.py:182} INFO - Started process (PID=2503) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:50:13,002] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 12:50:13,002] {logging_mixin.py:104} INFO - [2024-08-11 12:50:13,002] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:50:13,015] {logging_mixin.py:104} INFO - [2024-08-11 12:50:13,014] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 12:50:13,016] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 12:50:13,081] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.082 seconds
[2024-08-11 13:46:02,476] {scheduler_job.py:182} INFO - Started process (PID=2532) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:02,477] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 13:46:02,477] {logging_mixin.py:104} INFO - [2024-08-11 13:46:02,477] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:02,487] {logging_mixin.py:104} INFO - [2024-08-11 13:46:02,485] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 13:46:02,488] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:02,549] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 13:46:32,667] {scheduler_job.py:182} INFO - Started process (PID=2561) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:32,667] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 13:46:32,668] {logging_mixin.py:104} INFO - [2024-08-11 13:46:32,668] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:32,675] {logging_mixin.py:104} INFO - [2024-08-11 13:46:32,674] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 13:46:32,676] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 13:46:32,732] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.067 seconds
[2024-08-11 14:34:32,972] {scheduler_job.py:182} INFO - Started process (PID=2590) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:34:32,974] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:34:32,974] {logging_mixin.py:104} INFO - [2024-08-11 14:34:32,974] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:34:32,989] {logging_mixin.py:104} INFO - [2024-08-11 14:34:32,987] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:34:32,989] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:34:33,096] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.127 seconds
[2024-08-11 14:35:04,099] {scheduler_job.py:182} INFO - Started process (PID=2619) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:35:04,099] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:35:04,100] {logging_mixin.py:104} INFO - [2024-08-11 14:35:04,100] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:35:04,108] {logging_mixin.py:104} INFO - [2024-08-11 14:35:04,106] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:35:04,111] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:35:04,172] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 14:40:28,374] {scheduler_job.py:182} INFO - Started process (PID=2648) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:40:28,375] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:40:28,375] {logging_mixin.py:104} INFO - [2024-08-11 14:40:28,375] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:40:28,384] {logging_mixin.py:104} INFO - [2024-08-11 14:40:28,383] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:40:28,385] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:40:28,457] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.085 seconds
[2024-08-11 14:53:01,695] {scheduler_job.py:182} INFO - Started process (PID=2677) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:01,696] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:53:01,697] {logging_mixin.py:104} INFO - [2024-08-11 14:53:01,696] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:01,709] {logging_mixin.py:104} INFO - [2024-08-11 14:53:01,708] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:53:01,709] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:01,797] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.103 seconds
[2024-08-11 14:53:32,806] {scheduler_job.py:182} INFO - Started process (PID=2706) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:32,807] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:53:32,807] {logging_mixin.py:104} INFO - [2024-08-11 14:53:32,807] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:32,817] {logging_mixin.py:104} INFO - [2024-08-11 14:53:32,816] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:53:32,818] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:53:32,876] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 14:58:45,977] {scheduler_job.py:182} INFO - Started process (PID=2735) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:58:45,978] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 14:58:45,978] {logging_mixin.py:104} INFO - [2024-08-11 14:58:45,978] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:58:45,986] {logging_mixin.py:104} INFO - [2024-08-11 14:58:45,985] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 14:58:45,986] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 14:58:46,052] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.077 seconds
[2024-08-11 15:15:45,483] {scheduler_job.py:182} INFO - Started process (PID=2764) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:15:45,484] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 15:15:45,484] {logging_mixin.py:104} INFO - [2024-08-11 15:15:45,484] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:15:45,494] {logging_mixin.py:104} INFO - [2024-08-11 15:15:45,493] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 15:15:45,494] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:15:45,567] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.087 seconds
[2024-08-11 15:35:44,387] {scheduler_job.py:182} INFO - Started process (PID=2793) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:35:44,389] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 15:35:44,391] {logging_mixin.py:104} INFO - [2024-08-11 15:35:44,390] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:35:44,403] {logging_mixin.py:104} INFO - [2024-08-11 15:35:44,402] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 15:35:44,404] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:35:44,471] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.087 seconds
[2024-08-11 15:52:26,724] {scheduler_job.py:182} INFO - Started process (PID=2822) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:52:26,725] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 15:52:26,725] {logging_mixin.py:104} INFO - [2024-08-11 15:52:26,725] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:52:26,734] {logging_mixin.py:104} INFO - [2024-08-11 15:52:26,733] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 15:52:26,734] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 15:52:26,797] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.076 seconds
[2024-08-11 16:36:41,476] {scheduler_job.py:182} INFO - Started process (PID=2851) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:36:41,477] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 16:36:41,477] {logging_mixin.py:104} INFO - [2024-08-11 16:36:41,477] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:36:41,486] {logging_mixin.py:104} INFO - [2024-08-11 16:36:41,484] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 16:36:41,486] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:36:41,546] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.072 seconds
[2024-08-11 16:55:08,530] {scheduler_job.py:182} INFO - Started process (PID=2880) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:55:08,532] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 16:55:08,533] {logging_mixin.py:104} INFO - [2024-08-11 16:55:08,533] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:55:08,554] {logging_mixin.py:104} INFO - [2024-08-11 16:55:08,545] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 16:55:08,555] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 16:55:08,630] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.104 seconds
[2024-08-11 17:37:39,479] {scheduler_job.py:182} INFO - Started process (PID=2909) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:37:39,480] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 17:37:39,480] {logging_mixin.py:104} INFO - [2024-08-11 17:37:39,480] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:37:39,488] {logging_mixin.py:104} INFO - [2024-08-11 17:37:39,487] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 17:37:39,489] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:37:39,567] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.090 seconds
[2024-08-11 17:38:10,525] {scheduler_job.py:182} INFO - Started process (PID=2938) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:38:10,525] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 17:38:10,526] {logging_mixin.py:104} INFO - [2024-08-11 17:38:10,526] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:38:10,534] {logging_mixin.py:104} INFO - [2024-08-11 17:38:10,533] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 17:38:10,535] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 17:38:10,982] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.459 seconds
[2024-08-11 18:38:36,518] {scheduler_job.py:182} INFO - Started process (PID=2967) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:38:36,519] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 18:38:36,519] {logging_mixin.py:104} INFO - [2024-08-11 18:38:36,519] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:38:36,528] {logging_mixin.py:104} INFO - [2024-08-11 18:38:36,527] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 18:38:36,528] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:38:36,590] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 18:39:07,313] {scheduler_job.py:182} INFO - Started process (PID=2996) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:39:07,314] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 18:39:07,314] {logging_mixin.py:104} INFO - [2024-08-11 18:39:07,314] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:39:07,322] {logging_mixin.py:104} INFO - [2024-08-11 18:39:07,322] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 18:39:07,323] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 18:39:07,383] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 19:39:33,288] {scheduler_job.py:182} INFO - Started process (PID=3025) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:39:33,289] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 19:39:33,289] {logging_mixin.py:104} INFO - [2024-08-11 19:39:33,289] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:39:33,300] {logging_mixin.py:104} INFO - [2024-08-11 19:39:33,297] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 19:39:33,300] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:39:33,366] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.080 seconds
[2024-08-11 19:40:04,360] {scheduler_job.py:182} INFO - Started process (PID=3054) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:40:04,360] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 19:40:04,361] {logging_mixin.py:104} INFO - [2024-08-11 19:40:04,361] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:40:04,370] {logging_mixin.py:104} INFO - [2024-08-11 19:40:04,369] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 19:40:04,371] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 19:40:04,429] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 20:15:39,470] {scheduler_job.py:182} INFO - Started process (PID=3083) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:15:39,472] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:15:39,472] {logging_mixin.py:104} INFO - [2024-08-11 20:15:39,472] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:15:39,481] {logging_mixin.py:104} INFO - [2024-08-11 20:15:39,480] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:15:39,482] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:15:39,543] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 20:16:10,417] {scheduler_job.py:182} INFO - Started process (PID=3112) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:16:10,418] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:16:10,418] {logging_mixin.py:104} INFO - [2024-08-11 20:16:10,418] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:16:10,426] {logging_mixin.py:104} INFO - [2024-08-11 20:16:10,425] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:16:10,426] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:16:10,485] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.069 seconds
[2024-08-11 20:29:48,004] {scheduler_job.py:182} INFO - Started process (PID=3141) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:29:48,004] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:29:48,005] {logging_mixin.py:104} INFO - [2024-08-11 20:29:48,005] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:29:48,013] {logging_mixin.py:104} INFO - [2024-08-11 20:29:48,012] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:29:48,014] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:29:48,077] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 20:31:32,712] {scheduler_job.py:182} INFO - Started process (PID=3170) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:31:32,712] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:31:32,713] {logging_mixin.py:104} INFO - [2024-08-11 20:31:32,712] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:31:32,723] {logging_mixin.py:104} INFO - [2024-08-11 20:31:32,722] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:31:32,724] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:31:32,784] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 20:32:03,621] {scheduler_job.py:182} INFO - Started process (PID=3199) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:32:03,622] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:32:03,622] {logging_mixin.py:104} INFO - [2024-08-11 20:32:03,622] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:32:03,634] {logging_mixin.py:104} INFO - [2024-08-11 20:32:03,634] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:32:03,635] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:32:03,694] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 20:33:10,259] {scheduler_job.py:182} INFO - Started process (PID=3228) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:33:10,260] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:33:10,260] {logging_mixin.py:104} INFO - [2024-08-11 20:33:10,260] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:33:10,268] {logging_mixin.py:104} INFO - [2024-08-11 20:33:10,267] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:33:10,268] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:33:10,328] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 20:40:28,181] {scheduler_job.py:182} INFO - Started process (PID=3257) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:28,183] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:40:28,184] {logging_mixin.py:104} INFO - [2024-08-11 20:40:28,183] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:28,201] {logging_mixin.py:104} INFO - [2024-08-11 20:40:28,197] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:40:28,202] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:28,272] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.097 seconds
[2024-08-11 20:40:59,344] {scheduler_job.py:182} INFO - Started process (PID=3287) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:59,344] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:40:59,345] {logging_mixin.py:104} INFO - [2024-08-11 20:40:59,344] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:59,352] {logging_mixin.py:104} INFO - [2024-08-11 20:40:59,351] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:40:59,356] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:40:59,416] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 20:53:30,686] {scheduler_job.py:182} INFO - Started process (PID=3315) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:53:30,687] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:53:30,688] {logging_mixin.py:104} INFO - [2024-08-11 20:53:30,688] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:53:30,701] {logging_mixin.py:104} INFO - [2024-08-11 20:53:30,699] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:53:30,706] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:53:30,784] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.101 seconds
[2024-08-11 20:54:01,839] {scheduler_job.py:182} INFO - Started process (PID=3344) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:54:01,839] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 20:54:01,841] {logging_mixin.py:104} INFO - [2024-08-11 20:54:01,841] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:54:01,849] {logging_mixin.py:104} INFO - [2024-08-11 20:54:01,848] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 20:54:01,852] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 20:54:01,911] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 21:00:51,009] {scheduler_job.py:182} INFO - Started process (PID=3373) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:00:51,010] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:00:51,010] {logging_mixin.py:104} INFO - [2024-08-11 21:00:51,010] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:00:51,018] {logging_mixin.py:104} INFO - [2024-08-11 21:00:51,017] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:00:51,019] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:00:51,081] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.074 seconds
[2024-08-11 21:02:43,111] {scheduler_job.py:182} INFO - Started process (PID=3402) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:02:43,112] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:02:43,112] {logging_mixin.py:104} INFO - [2024-08-11 21:02:43,112] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:02:43,120] {logging_mixin.py:104} INFO - [2024-08-11 21:02:43,119] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:02:43,120] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:02:43,180] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 21:03:13,972] {scheduler_job.py:182} INFO - Started process (PID=3431) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:03:13,973] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:03:13,975] {logging_mixin.py:104} INFO - [2024-08-11 21:03:13,975] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:03:13,983] {logging_mixin.py:104} INFO - [2024-08-11 21:03:13,982] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:03:13,985] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:03:14,045] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 21:07:01,776] {scheduler_job.py:182} INFO - Started process (PID=3460) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:01,777] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:07:01,777] {logging_mixin.py:104} INFO - [2024-08-11 21:07:01,777] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:01,798] {logging_mixin.py:104} INFO - [2024-08-11 21:07:01,794] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:07:01,798] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:01,873] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.099 seconds
[2024-08-11 21:07:33,589] {scheduler_job.py:182} INFO - Started process (PID=3489) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:33,590] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:07:33,592] {logging_mixin.py:104} INFO - [2024-08-11 21:07:33,592] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:33,600] {logging_mixin.py:104} INFO - [2024-08-11 21:07:33,599] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:07:33,604] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:07:33,665] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.077 seconds
[2024-08-11 21:08:04,361] {scheduler_job.py:182} INFO - Started process (PID=3518) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:08:04,362] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 21:08:04,362] {logging_mixin.py:104} INFO - [2024-08-11 21:08:04,362] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:08:04,370] {logging_mixin.py:104} INFO - [2024-08-11 21:08:04,369] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 21:08:04,370] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 21:08:04,429] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
[2024-08-11 22:07:10,231] {scheduler_job.py:182} INFO - Started process (PID=3547) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:10,232] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 22:07:10,233] {logging_mixin.py:104} INFO - [2024-08-11 22:07:10,233] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:10,252] {logging_mixin.py:104} INFO - [2024-08-11 22:07:10,247] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 22:07:10,253] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:10,339] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.111 seconds
[2024-08-11 22:07:41,237] {scheduler_job.py:182} INFO - Started process (PID=3576) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:41,238] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 22:07:41,238] {logging_mixin.py:104} INFO - [2024-08-11 22:07:41,238] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:41,249] {logging_mixin.py:104} INFO - [2024-08-11 22:07:41,248] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 22:07:41,250] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:07:41,309] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.073 seconds
[2024-08-11 22:42:31,148] {scheduler_job.py:182} INFO - Started process (PID=3605) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:42:31,149] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 22:42:31,150] {logging_mixin.py:104} INFO - [2024-08-11 22:42:31,150] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:42:31,158] {logging_mixin.py:104} INFO - [2024-08-11 22:42:31,157] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 22:42:31,158] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:42:31,218] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 22:43:01,997] {scheduler_job.py:182} INFO - Started process (PID=3634) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:43:01,997] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 22:43:01,999] {logging_mixin.py:104} INFO - [2024-08-11 22:43:01,999] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:43:02,007] {logging_mixin.py:104} INFO - [2024-08-11 22:43:02,006] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 22:43:02,007] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 22:43:02,069] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 23:06:06,267] {scheduler_job.py:182} INFO - Started process (PID=3663) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:06,270] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 23:06:06,270] {logging_mixin.py:104} INFO - [2024-08-11 23:06:06,270] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:06,278] {logging_mixin.py:104} INFO - [2024-08-11 23:06:06,277] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 23:06:06,279] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:06,340] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.075 seconds
[2024-08-11 23:06:37,362] {scheduler_job.py:182} INFO - Started process (PID=3692) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:37,363] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 23:06:37,363] {logging_mixin.py:104} INFO - [2024-08-11 23:06:37,363] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:37,371] {logging_mixin.py:104} INFO - [2024-08-11 23:06:37,370] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 23:06:37,371] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:06:37,431] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.071 seconds
[2024-08-11 23:43:40,340] {scheduler_job.py:182} INFO - Started process (PID=3721) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:43:40,341] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 23:43:40,341] {logging_mixin.py:104} INFO - [2024-08-11 23:43:40,341] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:43:40,351] {logging_mixin.py:104} INFO - [2024-08-11 23:43:40,349] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 23:43:40,352] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:43:40,416] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.077 seconds
[2024-08-11 23:44:11,405] {scheduler_job.py:182} INFO - Started process (PID=3751) to work on /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:44:11,406] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/create_demo_table.py for tasks to queue
[2024-08-11 23:44:11,406] {logging_mixin.py:104} INFO - [2024-08-11 23:44:11,406] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:44:11,414] {logging_mixin.py:104} INFO - [2024-08-11 23:44:11,413] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/create_demo_table.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/create_demo_table.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2024-08-11 23:44:11,414] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/create_demo_table.py
[2024-08-11 23:44:11,474] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/create_demo_table.py took 0.070 seconds
